dance/transforms/normalize.py

```diff
@@ -575,9 +577,10 @@ class NormalizeTotal(AnnDataTransform):
 
     """
 
-    def __init__(self, exclude_highly_expressed_and_max_fraction: Optional[float] = None, **kwargs):
-        if exclude_highly_expressed_and_max_fraction < 0:
-            super().__init__(sc.pp.normalize_total, exclude_highly_expressed=False, **kwargs)
-        else:
-            super().__init__(sc.pp.normalize_total, exclude_highly_expressed=True,
-                             max_fraction=exclude_highly_expressed_and_max_fraction, **kwargs)
+    def __init__(self, max_fraction: Optional[float] = None, **kwargs):
```

Comment:
Register preprocessing function, see examples above. Otherwise it won't be correctly used by the pipeline functionality.

Reply:
I think this function is already registered, I will check again